sdk/program/src/log.rs

```diff
@@ -19,12 +19,37 @@ macro_rules! info {
             $arg4 as u64,
             $arg5 as u64,
         )
-    }; // `format!()` is not supported yet, Issue #3099
-       // `format!()` incurs a very large runtime overhead so it should be used with care
-       // ($($arg:tt)*) => ($crate::log::sol_log(&format!($($arg)*)));
+    };
+}
+
+/// Print a message to the log
+///
+/// There are two fast forms:
+/// 1. Single string: `msg!("hi")`
+/// 2. 5 integers: `msg!(1, 2, 3, 4, 5)`
+///
+/// The third form is more generic and incurs a very large runtime overhead so it should be used
+/// with care:
+/// 3. Generalized format string: `msg("Hello: 1, 2, {}", "World", 3)`
```

Comment:
Should this comment instead be: `msg!("Hello {}: 1, 2, {}", "World", 3)` ?

Reply:
Yep, thanks